sinonjs/samsam
# Updating version of lodash used

Fixes #60 

#### Purpose
Out of date and potentially vulnerable lodash version removed

#### How to verify
1. Check out this branch
2. Inspect package.json

#### Checklist for author

- [ y ] `npm run lint` passes
- [ n/a ] References to standard library functions are [cached](https://github.com/sinonjs/sinon/pull/1523).
